config/initializers/db2fog.rb

```diff
@@ -1,7 +1,11 @@
+require_relative 'spree'
+
 # See: https://github.com/yob/db2fog
 DB2Fog.config = {
     :aws_access_key_id     => Spree::Config[:s3_access_key],
     :aws_secret_access_key => Spree::Config[:s3_secret],
```

Comment:
Maybe we should just change these to `ENV['S3_ACCESS_KEY']` and `ENV['S3_SECRET']`?

Reply:
Then we don't need to require the other initializer, right?

Reply:
What about the changes made through the UI then?

Reply:
Well, with your proposal backups are completely separate from UI image-related settings, which I like it.

Reply:
Maybe we need to think about it more and decide: do we want two sets of keys for images and backups? Do we want the keys to be editable by instance managers from the UI if they're provisioned by ofn-install?

Reply:
IMO backups are a sysadmin responsibility and they should be controlled by `ofn-install` only. Having two separate sets of keys gives us that option at the expense of duplication for some instances.

Having said that, we know this works so I propose we merge, deploy it, get v2 out the door and then we can find a good solution without any rush.